sdk/push/src/main/java/com/klaviyo/push/KlaviyoPushService.kt

```diff
@@ -18,12 +18,11 @@ import com.klaviyo.coresdk.utils.KlaviyoPreferenceUtils
  */
 class KlaviyoPushService : FirebaseMessagingService() {
```

Comment:
did we want to make this open?

Reply:
I thought we wound up on no, but I'll check the discussion docs again.

Reply:
Yeah we landed on it being unnecessary to make our class open, when all they'd need to do is call the static methods from their implementation, see readme
### Manual implementation of `FirebaseMessagingService` (Advanced)
If you'd prefer to implement `FirebaseMessagingService` yourself, follow the FCM 
setup docs including referencing your own service class in the manifest.
Then update your implementation of `onNewToken` and `onMessageReceived` as below to communicate 
push tokens and notifications received to the Klaviyo SDK. Note that the launcher activity 
code snippets above are still required.
```kotlin
import com.google.firebase.messaging.RemoteMessage
import com.klaviyo.push.KlaviyoPushService

class YourPushService: FirebaseMessagingService() {
    override fun onNewToken(newToken: String) {
        super.onNewToken(newToken)
        KlaviyoPushService.setPushToken(newToken)
    }

    override fun onMessageReceived(message: RemoteMessage) {
        super.onMessageReceived(message)
        KlaviyoPushService.handlePush(message)
    }
}
```